docs/software/matlabinteractive.md

```diff
@@ -3,58 +3,225 @@ title: MATLAB Interactive
+
+> Note: Folders starting with a `.` are hidden. You can list them using `ls -a`. You don't have to put your virtual environment in a hidden folder -- you can call the folder whatever you want.
 
 ```
-[me@login01 ~]$ srun -p test -N 1 -n 8 --time=04:00:00 --pty --x11 bash
-[me@node01 ~]$ module load matlab/R2021a
-[me@node01 ~]$ matlab
-MATLAB is selecting SOFTWARE OPENGL rendering.
+[clusteruser@node16 ~]$ python -m venv .venv
+[clusteruser@node16 ~]$ source .venv/bin/activate
+(.venv) [clusteruser@node16 ~]$ pip install --upgrade pip
+(.venv) [clusteruser@node16 ~]$ pip install matlab-proxy
 ```
 
+And to re-enter the virtual environment later, just run:
+
+```
+[clusteruser@node16 ~]$ source .venv/bin/activate
+(.venv) [clusteruser@node16 ~]$ # now you are in the virtual environment
 ```
-[me@login01 ~]$ srun -p test -N 1 -n 2 --gres gpu:1 --time=04:00:00 --pty --x11 bash
-[me@gpu01 ~]$ module load matlab/R2021a
-[me@gpu01 ~]$ matlab
+
+> Note: Some developers prefer to manage packages and virtual enrionments using [conda](https://docs.conda.io/projects/conda/en/stable/) instead.
+
+From within the virtual environment run the `matlab-proxy-app`. You can find additional usage information [here](https://github.com/mathworks/matlab-proxy#usage).
+
+```
+(.venv) [clusteruser@node16 ~]$ env MWI_USE_EXISTING_LICENSE=True matlab-proxy-app
+
+INFO:MATLABProxyApp:Found MATLAB Executable: /export/matlab/Interactive/R2021a/bin/matlab with Root: /export/matlab/Interactive/R2021a
+INFO:MATLABProxyApp:
+================================
+
+   MATLAB can be accessed at:
+
+   http://localhost:37851
+
+================================
+INFO:MATLABProxyApp:!!! Launching MATLAB without providing any additional licensing information. This requires MATLAB to have been activated on the machine from which its being launched !!!
+INFO:MATLABProxyApp:Deleting any cached licensing files!
+INFO:MATLABProxyApp:Stopped (any running) MATLAB process.
 MATLAB is selecting SOFTWARE OPENGL rendering.
+
+                            < M A T L A B (R) >
+                  Copyright 1984-2021 The MathWorks, Inc.
+                  R2021a (9.10.0.1602886) 64-bit (glnxa64)
+                             February 17, 2021
+ 
+To get started, type doc.
+For product information, visit www.mathworks.com.
+
+>>
+```
+
+Take note of the port number in the access URL and what compute note you are running on. In this example it's port 37851 on node16. From your local computer, set up an ssh tunnel:
+
+```
+[localuser@localmachine ~]$ ssh -L 37851:node16:37851 login3.chpc.wustl.edu
```

Comment:
We should look to see if this is supported with `pam_slurm_adopt`

Reply:
I don't think this has anything to do with `pam_slurm_adopt`. That is needed for users to ssh into a compute node (i.e. ssh into node16 directly). It is not needed to forward ports on the login node.